libethcore/Transaction.cpp

```diff
@@ -127,8 +135,9 @@ void TransactionBase::streamRLP(RLPStream& _s, IncludeSignature _sig) const
 	else
 		_s << "";
 	_s << m_value << m_data;
+
 	if (_sig)
-		_s << (m_vrs.v + 27) << (u256)m_vrs.r << (u256)m_vrs.s;
+		_s << (m_vrs.v + m_chainId*2 + 35) << (u256)m_vrs.r << (u256)m_vrs.s;
```

Comment:
Please move this expression out to additional variable. Like `auto vOffset = m_chainId * 2 + 35`.